.github/workflows/ci.yml

```diff
@@ -47,10 +34,58 @@ jobs:
 
       - name: Install OPAM dependencies
         run: |
-          opam update
-          opam install -y dune
           opam uninstall -y brr
-          opam install -y . --deps-only --with-test
+          opam install -y . --deps-only --with-test || (opam update && opam install -y . --deps-only --with-test)
```

Comment:
The intention is to avoid (speed up) the `opam update` process if the version can be resolved within the `kxcteam/ocaml-general` package index, correct?

Reply:
yep. exactly.